### PR TITLE
Add `single_border_width` and `single_margin` to VerticalTile layout

### PR DESCRIPTION
**Pull request by luisalcarasr**
Add `single_border_width` and `single_margin` to the `VerticalTile` layout to be used when there is only one window.

### CODE REVIEW

File: libqtile/layout/verticaltile.py [hunk 1]
```diff
@@ -120,7 +122,16 @@ def configure(self, window, screen_rect):
...
+                border_width = self.single_border_width
+
+            # margin
+            if n > 1:
+                margin = self.margin
+            else:
+                if self.single_margin != None:
+                    margin = self.single_margin
+                else:
+                    margin = self.margin
```
Comment (elParaguayo) [inline libqtile/layout/verticaltile.py:134]:
```suggestion
            if n == 1 and self.single_margin is not None:
                margin = self.single_margin
            else:
                margin = self.margin
```

**Review by elParaguayo — CHANGES_REQUESTED** [review]
Looks good. Thanks for doing this.

I've suggested a change to tidy up the code a bit.

Can you also add a test for this?

File: libqtile/layout/verticaltile.py (hunk 1 above)
Comment (luisalcarasr) [reply libqtile/layout/verticaltile.py:134]: That looks better, Python is great. Thanks!

**Comment by elParaguayo** [comment]
Can you rebase this PR onto the latest master and force push again. We've fixed the CI issues so would be run CI on this.

Are you able to write a test or would you want some help with that?

**Comment by luisalcarasr** [comment]
@elParaguayo, I will try to write some tests for this, if I have problems I would appreciate the help.

**Comment by luisalcarasr** [comment]
Writing the tests I have detected an error with the `single_border_width`, the dimensions of the window are not recalculated. I'm working on the fix.

**Comment by luisalcarasr** [comment]
@elParaguayo, I've written the tests and fixed the bugs, let me know any other comments.

**Comment by luisalcarasr** [comment]
Should I add something to the `CHANGELOG`?
Is the name of the parameters correct? Some layouts use other names.

**Comment by luisalcarasr** [comment]
No longer needed, #3990 solved the requirement.

**Pull request closed without merging**